README.md

```diff
@@ -91,6 +83,11 @@ filtered_drds = search_transition.invoke do |builder|
 end
 ```
 
+You may also invoke transitions with automatic attribute and parameter matching
+```ruby
+drds.transitions['search'].invoke(search_term: '1812')
```

Comment:
Just to understand...are transitions names always strings?  (although, the invoke method supports symbols?)